documentation/api/io.strimzi.api.kafka.model.CruiseControlSpec.adoc

```diff
@@ -1,23 +1,62 @@
-// Module included in the following assemblies:
-//
-// assembly-cruise-control-concepts.adoc
+Configures a Cruise Control cluster.
 
-[id='ref-cruise-control-configuration-{context}']
-= Cruise Control configuration
+Configuration options relate to:
 
-The `config` property in `Kafka.spec.cruiseControl` contains configuration options as keys with values as one of the following JSON types:
+* Goals configuration
+* Capacity limits for resource distribution goals
+
+[id='property-cruise-control-config-{context}']
+=== `config`
+
+Use the `config` properties to configure Cruise Control options as keys.
+
+Standard Cruise Control configuration may be provided, restricted to those properties not managed directly by Strimzi.
+
+Configuration options that cannot be configured relate to the following:
+
+* Security (Encryption, Authentication, and Authorization)
+* Connection to the Kafka cluster
+* Client ID configuration
+* ZooKeeper connectivity
+* Web server configuration
+* Anomaly detection
+
+The values can be one of the following JSON types:
 
 * String
 * Number
 * Boolean
 
-You can specify and configure all the options listed in the "Configurations" section of the {CruiseControlConfigDocs}, apart from those managed directly by Strimzi.
-Specifically, you *cannot* modify configuration options with keys equal to or starting with one of the keys mentioned xref:type-CruiseControlSpec-reference[here].
-
-If restricted options are specified, they are ignored and a warning message is printed to the Cluster Operator log file.
-All the supported options are passed to Cruise Control.
-
-.An example Cruise Control configuration
+You can specify and configure the options listed in the {CruiseControlConfigDocs} with the exception of those options that are managed directly by Strimzi.
+Specifically, all configuration options with keys equal to or starting with one of the following strings are forbidden:
+
+* `bootstrap.servers`
+* `client.id`
+* `zookeeper.`
+* `network.`
+* `security.`
+* `failed.brokers.zk.path`
+* `webserver.http.`
+* `webserver.api.urlprefix`
+* `webserver.session.path`
+* `webserver.accesslog.`
+* `two.step.`
+* `request.reason.required`
+* `metric.reporter.sampler.bootstrap.servers`
+* `capacity.config.file`
+* `self.healing.`
+* `ssl.`
+* `kafka.broker.failure.detection.enable`
+* `topic.config.provider.class`
```

Comment:
Makes sense. I added the list to be consistent with the other `config` descriptions in the guide. Maybe we should do the same with those, like [KafkaClusterSpec schema reference](https://strimzi.io/docs/operators/in-development/configuring.html#type-KafkaClusterSpec-reference). Can we foresee any issue with that? Otherwise, I'll make this change elsewhere.

Reply:
replaced with: _See the description of the `config` property for the forbidden prefixes._

Reply:
Sounds great to me!